Automattic/wp-calypso
# Editor: Fix adding tags prefixed with spaces

Fixes #1836 (adding tags prefixed with spaces causes the API to discard them when saving the post).

wp-admin trims tag names when adding them, so I've taken the same approach here.

### To test

Follow repro steps at #1836.

Also verify that suggestions are still matched in the presence of leading spaces:

- Add a tag
- Press space
- Begin typing the name of an existing tag and verify that suggestions are still matched correctly despite the leading space
